src/runtime/storageNG/tests/handle-test.ts

```diff
@@ -64,75 +70,109 @@ async function getSingletonHandle(particle?: MockParticle):
   return handle;
 }
 
+let barType: EntityType;
+// tslint:disable-next-line variable-name
+let Bar: EntityClass;
+
+function newEntity(id: string) {
+  const bar = new Bar({});
+  Entity.identify(bar, id);
+  return bar;
+}
+
+async function containedIds(handle: CollectionHandle<Entity>): Promise<string[]> {
+  return (await handle.toList()).map(a => Entity.id(a));
```

Comment:
Just a simple FYI: you can return the promise here as-is, no need to await it.

Reply:
No, I need to map the conversion across the await'd list.

Reply:
Oh whoops, read that wrong